File: apiEZPZ/src/main/java/com/example/apiezpz/prohibited/repository/ProhibitedItemsRepository.java
package com.example.apiezpz.prohibited.repository;

import com.example.apiezpz.prohibited.domain.Prohibit;
import org.springframework.data.jpa.repository.JpaRepository;


import java.util.List;

public interface ProhibitedItemsRepository extends JpaRepository<Prohibit, Long> {
    List<Prohibit> findByGubun(String Gubun); //Gubun으로 데이터 조회
}

File: apiEZPZ/src/main/kotlin/com/example/apiezpz/prohibited/repository/ProhibitedItemsRepository.kt
package com.example.apiezpz.prohibited.repository

import com.example.apiezpz.prohibited.domain.Prohibit
import org.springframework.data.jpa.repository.JpaRepository

interface ProhibitedItemsRepository : JpaRepository<Prohibit, Long> {
    fun findByGubun(gubun: String): List<Prohibit> // Gubun으로 데이터 조회
} 